src/datamodules/imdb_datamodule.py

```diff
@@ -0,0 +1,61 @@
+import warnings
+import torch
+from pytorch_lightning import LightningDataModule
+from torchtext import data, datasets
+
+warnings.filterwarnings('ignore')
```

Comment:
Today I added a flag to main config for filtering all python warnings in the same way so I don't think this line is needed anymore